src/components/Board.vue

```diff
@@ -1,7 +1,14 @@
 <template>
   <div v-if="gameStats.gameStart">
     <h2>{{ msg }}</h2>
-    
+    <div class="wrapper">
+      <div v-for="(row, index) in gameSquares" :key="index" class="row">
```

Comment:
here is where I loop through the data for rows. Note the `(row, index)` portion. This is a standard option in Vue where I can name the current iteration, `row` in the first argument and I don't need parenthesis or a second argument. However in this case I want to access the `index` from the looping of rows, so I can add parenthesis and put in a second argument, which will always be a variable name for the `index`. In this case I called it `index` but I could have called it anything, like `i` or `idx`. Here I am being specifically verbose for learning purposes. More info can be found here: [https://vuejs.org/v2/guide/list.html](https://vuejs.org/v2/guide/list.html)

Vue warns that it is best practice to add a `v-bind:key` to elements that are subject to `v-for` loops. I am using the shorthand here, which is just `:key` - the `:` implies the `v-bind` directive. The `:key` needs to be unique throughout the loop so this is where I am using the `index`.